apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/CharSequenceAssertionSamples.kt

```diff
@@ -0,0 +1,232 @@
+//TODO remove file with 1.0.0
+@file:Suppress("DEPRECATION")
+
+package ch.tutteli.atrium.api.infix.en_GB.samples.deprecated
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.samples.fails
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class CharSequenceAssertionSamples {
+
+    @Test
+    fun containsBuilder() {
+        expect("ABC") contains o exactly 1 value "A"
+
+        expect("ABBC") contains o atLeast 2 value "B"
+
+        fails {
+            expect("AAAAAA") contains o atMost 3 value "A"
+        }
+    }
+
+    @Test
+    fun containsNotBuilder() {
+        expect("ABC") containsNot o value "X"
+
+        fails {
+            expect("ABC") containsNot o value "B"
+        }
+    }
+
+    @Test
+    fun contains() {
+        expect("ABC") contains o exactly 1 value "B"
+
+        expect("ABC123") contains values("AB", 'C', 12)
+
+        // holds because `contains` does not search for unique matches
+        // use `contains o exactly 2 value "A"` to check if subject contains two "A"s
+        expect("ABC") contains o exactly 1 the values("A", "A")
+
+        fails {
+            expect("ABC") contains o exactly 1 value  "X"
+        }
+
+        fails { // because subject does not contain all values
+            expect("ABC") contains o exactly 1 the values("A", 99)
+        }
+    }
+
+    @Test
+    fun containsNot() {
+        expect("ABC") containsNot o value "X"
```

Comment:
same here, use the shortcut and not the builder
```suggestion
        expect("ABC") containsNot "X"
```